README.md

```diff
@@ -68,6 +68,18 @@ This will create a VPC, Application Load Balancer, an Amazon ECS Service with th
 
 Once you're finished playing around with this project, you can delete it and all the AWS resources associated it by running `copilot app delete`.
 
+## Running a one off task 🏃‍♀️
```

Comment:
Should we change this to "one-off" with a hyphen?

Reply:
Done!! Thank you!